src/server.jl

```diff
@@ -383,7 +369,12 @@ function ws_tracker(ws::HTTP.WebSockets.WebSocket, target::AbstractString)
         # - We therefore do not propagate the error but merely store the information
         # that there was a forcible interruption of the websocket so that the
         # interruption can be guaranteed to be propagated.
-        WS_INTERRUPT[] = true
+        if !WebSockets.isok(err)
```

Comment:
I'm a little unsure about including this change because as far as I understand, we're never actually _reading_ from the websocket connections we make? If that's indeed the case, I don't think we'll ever have the case where a websocket CLOSE error will be thrown by receiving it from the remote (since we're never "reading" messages).

But I also think this is harmless because `WebSockets.isok` will only return true for those specific non-error websocket closures, so it shouldn't hurt to include this.